UI Import Data

lposo opened the pull request:
Created a basic home page. On the bottom is the box where users will import their data to. No functionality has been added to the box.

Mehrdadmaskull committed:
Delete .DS_Store

Commit:
Fixing gitignore

Commit:
Temporary Header

Added blue header to app.js. TEsting out components.

Commit:
Added Splash/Blurb

Added splash/blurb. Used a temporaty image for the splash art.

Commit:
Changed fonts, added new splash.

Commit:
Added box with upload button

Added a box where users will go to when they wish to upload data. No functionality to the button so far.

lposo committed:
Merge branch 'develop' into ui-import-data

GD-Dheer commented on Client/src/Components/DataCell.js:
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+
```
I would add a function here to make an empty call to the upload API in backend

GD-Dheer commented on Client/src/Components/DataCell.js:
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Isnt this supposed to be on a new page? The upload is gonna be on the first page?

lposo replied:
It's just temporary for this sprint. Later on, once we have the ability to switch between pages it'll be moved to its own page.

GD-Dheer commented on Client/src/Components/Blurb.js:
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```
Design consideration: You could make one component of the box and just reuse it to create the others. This could be useful & creates re-usable component depending on how the other pages look like.

Is it possible to get it the blurbs aligned?

GD-Dheer commented on Client/src/Components/Blurb.js:
```diff
@@ -0,0 +1,34 @@
+            </div>
+            <div className = "SecondBlurb">
+                <h1>Downloads</h1>
+                <p>Downloads have never been so easy. Select the datasets you want, choose the format you need, preview for perfection and DOWNLAOD WITH A CLICK.</p>
```
Typos..

GD-Dheer commented on Client/src/App.js:
```diff
@@ -1,7 +1,10 @@
+import DataCell from "./Components/DataCell.js"
 function App() {
   const[apiResponse, setApiResponse] = useState("");
 
```
Could you update the directory structure and create a directory for each component and put the js & css file in there? The way it looks now it will get cluttered really quickly.

GD-Dheer reviewed (commented):
Will wait for input - hopefully gets resolved today. 
React still has its learning curve and you got the code to match the UI, nicely done.

Commit:
Fixed code based on recommendations.

Reorganized Components, Fixed Typos, Changed Blurb to use Box, and added an empty call to api.

Commit:
Merge branch 'ui-import-data' of https://github.com/Data-Boom/SOEN490 into ui-import-data

GD-Dheer commented on Server/src/app.ts:
```diff
@@ -33,6 +33,10 @@ app.get('/note', function(req,res) {
   res.status(200).json("L, did you know Shinigami love apples?");
 });
 
+app.get('dataupload', function(req,res){
```
Perfect this will be updated when the routing is setup!
